### DISCUSSION TITLE
Retest is result is more then a user set percentage than last result.

### DISCUSSION THREAD

**Discussion by bobluck** [alexjustesen/speedtest-tracker#1016] [Ideas]
Hi, Love the app. I would like to make a suggestion that an automatic retest is run if the result deviates more than a user set percentage from the last result prior to recording it. 

Say average speed is 200 but the latest test comes in at 50 or 150, rerun a test using a different random server prior to posting the well below average result. Or require a second opinion prior to recording large change from average.

Thanks for the great app.

**Comment by alexjustesen** [comment]
Nice idea, I'll get that into the queue for when I rewrite tests.

**Comment by bobluck** [comment]
Thanks
